### ISSUE TITLE
EndpointURL not working in new version

### ISSUE THREAD

**Issue by nenti** [apollographql/graphql-server#360]
I recently updated graphql-server-lambda to version 0.7.0 and get the following error on `graphiql` endpoint:
```
{
  "statusCode": 404,
  "error": "Serverless-offline: route not found.",
  "currentRoute": "post - /graphql",
  "existingRoutes": [
    "post - /products",
    "get - /products/graphiql"
  ]
}
```

My handler looks as out of the documentation:
```
exports.graphiqlHandler = server.graphiqlLambda({
    endpointURL: '/products'
})
```

Is anyone experiencing the same problem with the new version? The version before worked fine. It seems as if my config is overwritten with the default value.

**Comment by Ephem** [comment]
I had the same problem and tracked it down to https://github.com/apollographql/graphql-server/blob/v0.7.0/packages/graphql-server-module-graphiql/src/renderGraphiQL.ts#L124 where indeed `/graphql` is used directly. This seems to be fixed in 0.7.1 though and upgrading fixed my problems! :)

**Comment by helfer** [comment]
Thanks for filing the issue. This should indeed be fixed in 0.7.1. Sorry it slipped through review!
